models/migrations/v51.go

```diff
@@ -1 +1,61 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
 package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/modules/setting"
+	"github.com/go-xorm/xorm"
+	"time"
```

Comment:
wrong import order

Reply:
Which one would be right? This is formatted by `make fmt`

Reply:
see https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
Done.